front/app/components/UI/GanttChart/utils.test.ts

```diff
@@ -0,0 +1,160 @@
+import {
+  getTimeRangeDates,
+  groupDayCellsByMonth,
+  groupWeekCellsByYear,
+  getDayCells,
+  getWeekCells,
+  getQuarterCells,
+  getPreciseOffsetInMonths,
+  getPreciseDurationInMonths,
+  getPreciseOffsetInWeeks,
+  getPreciseDurationInWeeks,
+  getPreciseOffsetInQuarters,
+  getPreciseDurationInQuarters,
+  dayWidth,
+  weekWidth,
+} from './utils';
+
+// Helper to create dates and avoid timezone issues in tests.
+const date = (str: string) => new Date(`${str}T00:00:00`);
```

Comment:
And maybe here as well?  This way the test might fail if I run it from a timezone west of GMT